tests/acceptance/features/coreApiWebdavProperties/copyFile.feature

```diff
@@ -735,6 +757,26 @@ Feature: copy file
       | old              |
       | new              |
 
+
+  Scenario Outline: copying a file out of a shared folder shared via secure viewer permissions as the sharee
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Brian" has created folder "/testshare"
+    And user "Brian" has uploaded file with content "hello world" to "testshare/fileInsideShare.txt"
+    And user "Brian" has sent the following resource share invitation:
+      | resource        | testshare     |
+      | space           | Personal      |
+      | sharee          | Alice         |
+      | shareType       | user          |
+      | permissionsRole | Secure viewer |
+    When user "Alice" copies file "/Shares/testshare/fileInsideShare.txt" to "/fileInsideShare.txt" using the WebDAV API
+    Then the HTTP status code should be "500"
```

Comment:
i don't think this is the expected status code `500` ?

Reply:
This table summarizes my finding
https://github.com/owncloud/ocis/issues/9334#issuecomment-2182573751

Reply:
I see according to the issue comment it should be 403. So better but it 403 and put it in blocked state until it get fixed.